src/main/java/ita/softserve/course_evaluation/service/impl/AnswerToFeedbackServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package ita.softserve.course_evaluation.service.impl;
+
+import ita.softserve.course_evaluation.dto.AnswerDto;
+import ita.softserve.course_evaluation.dto.AnswerDtoMapper;
+import ita.softserve.course_evaluation.repository.AnswerToFeedbackRepository;
+import ita.softserve.course_evaluation.service.AnswerToFeedbackService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class AnswerToFeedbackServiceImpl implements AnswerToFeedbackService {
+    private final AnswerToFeedbackRepository answerRepository;
+
+    @Autowired
+    public AnswerToFeedbackServiceImpl(AnswerToFeedbackRepository answerRepository) {
+        this.answerRepository = answerRepository;
+    }
+
+    @Override
+    public List<AnswerDto> getAllAnswer() {
+        return AnswerDtoMapper.toDto(answerRepository.findAll());
+    }
+
+    @Override
+    public AnswerDto saveAnswer(AnswerDto answer) {
+        answerRepository.save(AnswerDtoMapper.fromDto(answer));
+        return answer;
+    }
+
+    @Override
+    public AnswerDto findAnswerById(long id) {
+        return AnswerDtoMapper
+                .toDto(answerRepository
+                        .findById(id)
+                        .orElseThrow(() -> new RuntimeException("Answer was not found for id: " + id)));
```

Comment:
It's better to create a specific exception, like AnswerNotFoundException, but I believe if there is no answer in DB - we get ```null```